scripts/test_version.kts

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env kscript
```

Comment:
Add apache license header on all new .kt files

Reply:
This is deleted.